doc/translations/ar.po

```diff
@@ -31113,7 +31113,7 @@ msgid ""
 "select the item in question directly. Both of these actions will be reset to "
 "the beginning of the list if the timeout duration has passed since the last "
 "keystroke was registered. You can adjust the timeout duration by changing "
-"[member ProjectSettings.gui/timers/incremental_search_max_interval_msec]."
+"[member ProjectSettings.gui/timers/incremental_search_max_interval]."
```

Comment:
The `.po` and `.pot` files in the `doc/translations/` folder are auto-generated when syncing translations from Weblate, you should not manually change them.